### ISSUE TITLE
L029 false negative (i.e., generates an error on code that is correct) - maybe

### ISSUE THREAD

**Issue by EdmundsEcho** [sqlfluff/sqlfluff#1349]
## Expected Behaviour

input:
```sql
alter table project_link_tokens
add constraint fk_project_tokens_tokens  -- <<< line that generates the lint
foreign key ( token_id ) references tnc.tokens( id );
```
output: nothing

## Observed Behaviour
output:
```fish
[sqlfluff L029] [E] Keywords should not be used as identifiers.
```

## Dialect
postgres

## Version
0.6.4

## Configuration
```
[sqlfluff]
dialect = postgres
output_line_length = 80
capitalisation_policy = lower
```

**Comment by WittierDinosaur** [comment]
I've just tested this, and it'll be resolved in the next release of sqlfluff. The current version of main parses this statement correctly.

**Comment by EdmundsEcho** [comment]
Great!  You said "main" parses it correctly which to me means I don't have the latest version.  What's the difference between "next release" and the main branch?

**Comment by WittierDinosaur** [comment]
I’m new here, so don’t take my word as gospel, but I believe a release is pushed to pypi monthly, and the last release was the 20th August. So your pip install reflects the state of the code then, and not main now

**Comment by tunetheweb** [comment]
It's manual, so its' roughly monthly. Or whenever we feel like there's enough release-worthy changes. Definitely not automatic on merge to `main` anyway.

@EdmundsEcho are you implying that you've downloaded locally and tested on `main` and it's NOT fixed for you? Or were you assuming that once it was merged to `main` it was available in a new release?
